internal/config/file_classification.go

```diff
@@ -11,7 +11,7 @@ type fileClassification struct {
 }
 
 func (cfg fileClassification) loadDefaultValues(v *viper.Viper) {
-	v.SetDefault("file-classification.cataloger.enabled", catalogerEnabledDefault)
```

Comment:
File classifiers aren't that expensive and probably should be removed or enabled by default. This PR attempts to start a general `binary` cataloger